.github/workflows/process-load-test-result.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           while [ "$checkCount" != "3" ]
           do
             sleep 20
-            checkCount=$(gh pr status --jq '[.currentBranch .statusCheckRollup[] | select((.conclusion=="SUCCESS") and ((.name=="Build on Ubuntu") or (.name=="Build on Windows") or (.name=="codecov/project")))]|length' --json statusCheckRollup)
+            checkCount=$(gh pr status --jq '[.currentBranch .statusCheckRollup[] | select((.conclusion=="SUCCESS") and ((.name=="Build on Ubuntu") or (.name=="Build on Windows") or (.name=="codecov/project")))] | select(.context!="license/cla") | length' --json statusCheckRollup)
```

Comment:
excluding `license/cla` is not needed since we are explicitly checking using the status name

Reply:
Fixed in [a330c06](https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/156/commits/a330c069411a7cd1b276a5c0a7e38df7ace64fc6)